webpack.config.js

```diff
@@ -15,28 +15,34 @@ const config = {
   },
 
   module: {
-    loaders: [
+    rules: [
       {
         test: /\.scss$/,
-        loader : 'style!css!sass!postcss'
+        use : ['style-loader','css-loader', 'sass-loader', 'postcss-loader']
```

Comment:
This is a little pedantic and I realize that we don't have a JavaScript linter 😓 .
Could you kindly adjust the spacing so that it's consistent? Goes for the `elm` loaders as well. 🙂 

The rest of it looks good to me. 👍